<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportRobometMontageDialog</class>
 <widget class="QDialog" name="ImportRobometMontageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Robomet Montage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="2" column="2">
    <widget class="QLineEdit" name="originY">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="originZ">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="RobometListWidget" name="robometListWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="dataDisplayTypeCB">
     <item>
      <property name="text">
       <string>Outline Only</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Individual Tiles</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Stitched Montage</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="spacingZ">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="originX">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QLineEdit" name="montageNameLE">
     <property name="text">
      <string>Untitled Montage</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Data Display Type</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="spacingY">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="spacingX">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="changeSpacingCB">
     <property name="text">
      <string>Override Spacing (X, Y, Z)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>Montage Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="changeOriginCB">
     <property name="text">
      <string>Override Origin (X, Y, Z)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RobometListWidget</class>
   <extends>QWidget</extends>
   <header location="global">RobometListWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImportRobometMontageDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImportRobometMontageDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
